dom5.ts

```diff
@@ -12,13 +12,10 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-/// <reference path="./custom_typings/main.d.ts" />
 import * as cloneObject from 'clone';
-import * as parse5 from 'parse5';
-import {ASTNode as Node} from 'parse5';
-export {ASTNode as Node} from 'parse5';
```

Comment:
*@rictic responded:*

(arg, can't reply to the thread on ASTNode vs Node because github)

I don't have a strong opinion on ASTNode vs Node really. Not adding another level of indirection is probably best, but easiest thing would probably be first ceasing to use it, then removing it here.